warning: catching polymorphic type ‘class std::invalid_argument’ by value [-Wcatch-value=]  catch (std::invalid_argument)

I got this error during the `srt-live-transmit` compilation on Ubuntu 18.10 running kernel: `4.18.0-15-generic` it is just a warning and after all the package managed to compile. This is an excerpt of the warning: 
```
Scanning dependencies of target srt-live-transmit
[ 97%] Building CXX object CMakeFiles/srt-live-transmit.dir/apps/srt-live-transmit.cpp.o
/home/gstoyanov/srt/apps/srt-live-transmit.cpp: In function ‘int main(int, char**)’:
/home/gstoyanov/srt/apps/srt-live-transmit.cpp:229:17: warning: catching polymorphic type ‘class std::invalid_argument’ by value [-Wcatch-value=]
     catch (std::invalid_argument)
                 ^~~~~~~~~~~~~~~~
[100%] Linking CXX executable srt-live-transmit
```

--- a/apps/srt-live-transmit.cpp
+++ b/apps/srt-live-transmit.cpp
@@ -226,7 +226,7 @@ int main( int argc, char** argv )
         transmit_bw_report = stoul(Option("0", "r", "report", "bandwidth-report", "bitrate-report"));
         transmit_stats_report = stoul(Option("0", "s", "stats", "stats-report-frequency"));
     }
-    catch (std::invalid_argument)
+    catch (std::invalid_argument &)
     {
         cerr << "ERROR: Incorrect integer number specified for an option.\n";
         return 1;

--- a/testing/srt-test-live.cpp
+++ b/testing/srt-test-live.cpp
@@ -306,7 +306,7 @@ int main( int argc, char** argv )
         transmit_stats_report = stoi(Option("0", "s", "stats", "stats-report-frequency"));
         stoptime = stoul(Option("0", "d", "stoptime"));
     }
-    catch (std::invalid_argument)
+    catch (std::invalid_argument &)
     {
         cerr << "ERROR: Incorrect integer number specified for an option.\n";
         return 1;

